node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -167,10 +167,6 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       descr = "Start a stand-alone node."
     )
 
-    val devMode = opt[Flag](
-      descr = "Enable all developer tools."
```

Comment:
why remove here?

Reply:
@zsluedem it is added in the bottom if the file next to deployerPrivateKey option